template/metadata-JHU_IDD-CovidSP.txt

```diff
@@ -0,0 +1,54 @@
+team_name: Johns Hopkins ID Dynamics COVID-19 Working Group
```

Comment:
I thought we were going with the json format as in the US? Happy to discuss if you think this is more straightforward.

Reply:
I had it wrong - format is YAML so this makes sense.